packages/server/src/api/controllers/role.ts

```diff
@@ -106,6 +106,21 @@ export async function save(ctx: UserCtx<SaveRoleRequest, SaveRoleResponse>) {
   )
   role._rev = result.rev
   ctx.body = role
+
+  const devDb = context.getDevAppDB()
+  const prodDb = context.getProdAppDB()
+
+  if (await prodDb.exists()) {
+    const replication = new dbCore.Replication({
+      source: devDb.name,
+      target: prodDb.name,
+    })
+    await replication.replicate({
+      filter: (doc: any, params: any) => {
```

Comment:
Wondering if we should just always replicate all roles here - same effort but would make sure roles are just always consistently in sync, any role operation makes sure all roles get synced (like for example, if roles depended on each other and some were updated at the same time).

Reply:
It's easy to do. I'm struggling to come up with a reason _not_ to do it. I'll defer to your judgement.